README.md

```diff
@@ -1,69 +1,88 @@
-}
 
+    // Get player for video with id `gC6dQrScmHE`
+    let player = player(
+        String::from("gC6dQrScmHE"), // Video ID
+        String::from(""), // Parameters
+        &client_config // Client config
+    ).await.except("Couldn't fetch the video info");
+
+    println!("{}", player.video_details.title); // Print the video title
+}
 ```
-# With logging
-For logging tracing is used so tracing_subscribe can be installed for easier use of tracing. The library has as target youtubei_rs with debug,trace and error levels.
+
+### With logging
+For logging, tracing is used so `tracing_subscribe` can be installed for easier use of tracing. The library has as target youtubei_rs with debug, trace, and error levels.
+
 ```rust
-use youtubei_rs::{query::player, utils::default_client_config};
+use youtubei_rs::{query::player, utils::default_client_config, tokio};
+
 #[tokio::main]
 async fn main() {
     std::env::set_var("RUST_LOG", "youtubei_rs=debug");
     tracing_subscriber::fmt::init();
-    // create default client_config with WEB client
+
+    // Create the default client_config with web client
     let client_config = &default_client_config();
-    // get player for video with id gC6dQrScmHE
-    let player: PlayerResult = player(String::from("gC6dQrScmHE"),String::from(""),&client_config).await.unwrap();
-    println!("{}",player.video_details.title); // video title
-}
 
+    // Get player for video with id `gC6dQrScmHE`
+    let player = player(
+        String::from("gC6dQrScmHE"), // Video ID
+        String::from(""), // Parameters
+        &client_config // Client config
+    ).await.except("Couldn't fetch the video info");
+
+    println!("{}", player.video_details.title); // Print the video title
+}
 ```
-# Supported queries (legacy)
-- get_video: Fetches all information about the video except captions and storyboards
-- get_channel_info: Fetches all channel information and about tab
-- get_channel_tab: Fetches a specific tab like videos to get channel videos
-- search: Search youtube
-- load_search: Continue search with ctoken
-- get_comments: Loads initial comments or more comments for video
-- load_related_videos: Loads more related videos
-- get_playlist: Loads a playlist
-
-For more in depth info take a look at [query.rs](https://github.com/11Tuvork28/youtubei-rs/blob/main/src/query.rs) and [tests.rs](https://github.com/11Tuvork28/youtubei-rs/blob/master/src/tests.rc)
\ No newline at end of file
+
+## Legacy queries
+ - `get_video`: Fetches all information about the video except captions and storyboards
+ - `get_channel_info`: Fetches all channel information and about tab
+ - `get_channel_tab`: Fetches a specific tab like videos to get channel videos
+ - `search`: Search YouTube
+ - `load_search`: Continue search with ctoken
+ - `get_comments`: Loads initial comments or more comments for video
+ - `load_related_videos`: Loads the related videos
+ - `get_playlist`: Loads a playlist
+
```

Comment:
Could you while you're at it remove the legacy queries part since nobody should be using those, would be nice

Reply:
Should I remove all legacy queries? Are they going to be removed from the codebase?

Reply:
Yeah all, since they are already removed in dev but not main(released)